dart/lib/flat_buffers.dart

```diff
@@ -1241,3 +1246,72 @@ class _VTable {
     }
   }
 }
+
+// Allocator interface. This is flatbuffers-specific - only for [Builder].
+abstract class Allocator {
+  ByteData allocate(int size);
```

Comment:
needs a doc comment. Also consider creating our own buffer interface rather than directly using ByteData.

Reply:
Added the docs for now. I'll check about having a custom buffer.